transformers/transformWarframe.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const POLARITIES = {
+  Bar: 'naramon',
+  V: 'madurai',
+  D: 'vazarin',
+};
+
+const transformPolarities = ({ Polarities, AuraPolarity }, targetWeapon) => {
+  const outputFrame = { ...targetWeapon };
+  if (AuraPolarity) {
+    outputFrame.AuraPolarity = POLARITIES[AuraPolarity];
+  }
+  if (Polarities) {
+    outputFrame.polarities = Polarities.map(polarity => POLARITIES[polarity]);
+  } else {
+    outputFrame.polarities = [];
+  }
+  return outputFrame;
+};
+
+const transformWarframe = (oldFrame, imageUrls) => {
+  let newFrame;
+  if (!oldFrame || !oldFrame.Name) {
+    return undefined;
+  }
+  try {
+    const {
+      Armor,
+      AuraPolarity,
+      Conclave,
+      Energy,
+      Health,
+      Image,
+      Mastery,
+      Polarities,
+      Shield,
+      Sprint,
+      Introduced,
+      Sex,
+      // Vaulted,
+    } = oldFrame;
+    const { Name } = oldFrame;
```

Comment:
I'm not seeing any mechanism to grab the profile vids & prime trailers, which would be needed for parity with [the existing flatfile](https://api.warframestat.us/warframes), as well as location and a summarized color.

- [ ] profile video
- [ ] prime trailer
- [ ] combining prime into non-prime _optional_
- [ ] color ([example](https://github.com/WFCD/warframe-nexus-query/blob/master/lib/AttachmentCreator.js))
- [ ] location